src/index.ts

```diff
@@ -1,21 +1,30 @@
-import { SwaggerObject } from './swagger';
+import { SwaggerObject, TSwaggerObject } from './swagger';
 import * as prettier from 'prettier';
-import { map, write } from './fs';
+import { map, TFSEntity, write } from './fs';
 import { TSerializer } from './utils';
 import * as fs from 'fs-extra';
-import { fromNullable } from 'fp-ts/lib/Option';
+import { fromNullable, Option } from 'fp-ts/lib/Option';
 import * as path from 'path';
 import { ThrowReporter } from 'io-ts/lib/ThrowReporter';
 import { PathReporter } from 'io-ts/lib/PathReporter';
-import { last } from 'fp-ts/lib/Array';
+import { head, last } from 'fp-ts/lib/Array';
+import { Either, Right } from 'fp-ts/lib/Either';
+import { ValidationError } from 'io-ts';
+import { serialize } from './language/typescript';
 
 const log = console.log.bind(console, '[SWAGGER-CODEGEN-TS]:');
 
+type TJSON = {
+	[key: string]: unknown;
+};
+
+export type TFileReader = (buffer: Buffer) => TJSON;
+
 export type TGenerateOptions = {
 	/**
-	 * Path to json spec
+	 * Paths to spec files
 	 */
-	pathToSpec: string;
+	pathsToSpec: string[];
```

Comment:
How does multifile support relate to yaml format?

Reply:
see the previous PR first